cockpit/endpoint/cockpit_endpoint.py

```diff
@@ -20,3 +24,13 @@ def get_db():
 async def list_apps(db: Session = Depends(get_db)):
 
     return JSONResponse(status_code=status.HTTP_200_OK, content="69")
+
+
+@router.put("/add", tags=["Backend AppStore"])
```

Comment:
I think it should be @router.post and tags = "Backend Cockpit" instead of AppStore ;)